game_api.txt

```diff
@@ -363,14 +371,27 @@ The farming API allows you to easily register plants and hoes.
 ### Plant definition
 
 	{
-		description = "",                      -- Description of seed item
-		harvest_description = "",              -- Description of harvest item
-		                                       -- (optional, derived automatically if not provided)
-		inventory_image = "unknown_item.png",  -- Image to be used as seed's wield- and inventory image
-		steps = 8,                             -- How many steps the plant has to grow, until it can be harvested
-		-- ^ Always provide a plant texture for each step, format: modname_plantname_i.png (i = stepnumber)
-		minlight = 13,                         -- Minimum light to grow
-		maxlight = default.LIGHT_MAX           -- Maximum light to grow
+		-- Description of seed item
+		description = "My Plant",
+
+		-- Description of harvest item (optional, derived automatically if not
+		-- provided)
+		harvest_description = "",
+
+		-- Image to be used as seed's wield- and inventory image
+		inventory_image = "unknown_item.png",
+
+		-- How many steps the plant has to grow, until it can be harvested
+		steps = 8,
+		-- ^ Always provide a plant texture for each step, format:
+		-- modname_plantname_i.png (i = stepnumber)
+
+		-- Groups of soil nodes on which the seed can grow
+		fertility = {"grassland"},
+
+		-- Minimum and maximum light to grow
+		minlight = 1,
+		maxlight = default.LIGHT_MAX
```

Comment:
The plant definition changes above make the documentation longer and slightly less readable (because everything is so spaced out with comments inbetween). I prefer the old format, as long as `^ Always provide a plant texture for each step, format: modname_plantname_i.png (i = stepnumber)` is aligned with the comments above.